cmd/lotus-miner/actor.go

```diff
@@ -305,6 +306,78 @@ var actorWithdrawCmd = &cli.Command{
 	},
 }
 
+var beneficiaryWithdrawCmd = &cli.Command{
+	Name:      "beneficiary-withdraw",
+	Usage:     "withdraw available balance to beneficiary address. Will not work from non-beneficiary owner addresses. Use lotus-miner for non-beneficiary owner withdrawals",
```

Comment:
This helptext  is confusing me -- what are you trying to say here? I think what you need to tell the user is that this command will try to _send_ the message from the beneficiary address. If one isn't set, or if they just want to use the owner address, they should instead use `lotus-miner withdraw`?